tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -392,6 +385,218 @@ TestParticipantRecordListing:
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: contain
+          Var: TAB_CLASS
+          Value: 'active'
+  # Check Providers tab and content
+    - Type: click
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.providers_tab_nav$
+    - Type: get_attribute
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.providers_tab_nav$
+      Greps:
+        - var: TAB_CLASS
+          attr: class
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: contain
+          Var: TAB_CLASS
+          Value: 'active'
+    - Type: scroll_to
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.health_insurance_button$
+    - Type: scroll_to
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.secondary_health_insurance_button$
+    - Type: scroll_to
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.tertiary_health_insurance_button$
+    - Type: scroll_to
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.primary_care_physician$
+    - Type: scroll_to
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.pharmacy_button$
+    - Type: scroll_to
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.preferred_lab_button$
+    - Type: scroll_to
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.specialist_button$
+  #Check Call History tab
+    - Type: click
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.call_history_tab_button$
+    - Type: get_attribute
+      Strategy: xpath
+      Role: desktopChrome
+      Id: $PAGE.care_platform_participant_details.call_history_tab_button$
+      Greps:
+        - var: TAB_CLASS
+          attr: class
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: contain
+          Var: TAB_CLASS
+          Value: 'active'
+  #Go to follow-up tickets tab
+    - Type: click
+      Strategy: xpath
+      Role: desktopChrome
+      Id: $PAGE.care_platform_participant_details.follow_up_tab_button$
+    - Type: get_attribute
+      Role: desktopChrome
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.follow_up_tab_button$
+      Greps:
+        - var: TAB_CLASS
+          attr: class
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: contain
+          Var: TAB_CLASS
+          Value: 'active'
```

Comment:
please add a tear down to hang up the call

Reply:
added, even though is not included in the test, is the better finish for the environment and other tests